agent/config/config_test.go

```diff
@@ -101,9 +101,8 @@ func TestEnvironmentConfig(t *testing.T) {
 	assert.True(t, conf.TaskIAMRoleEnabled, "Wrong value for TaskIAMRoleEnabled")
 	assert.True(t, conf.TaskIAMRoleEnabledForNetworkHost, "Wrong value for TaskIAMRoleEnabledForNetworkHost")
 	assert.True(t, conf.ImageCleanupDisabled, "Wrong value for ImageCleanupDisabled")
-	assert.True(t, conf.TaskCPUMemLimit, "Wrong value for TaskCPUMemLimit")
-	assert.True(t, conf.TaskENIEnabled, "Wrong value for TaskNetwork")
 
+	assert.True(t, conf.TaskENIEnabled, "Wrong value for TaskNetwork")
```

Comment:
What happened to the assertion on `conf.TaskCPUMemLimit`?

Reply:
It isn't set at this point because it isn't set until merged with the default.